moto/batch/models.py

```diff
@@ -1593,7 +1573,7 @@ def list_jobs(
 
         job_queue = self.get_job_queue(job_queue_name)
         if job_queue is None:
-            raise ClientException("Job queue {0} does not exist".format(job_queue))
+            raise ClientException(f"Job queue {job_queue_name} does not exist")
```

Comment:
Bugfix - we know that 'job_queue' is None, so returning the name instead makes more sense